error: use of undeclared identifier '_1

Can't build 0.4.10 freeorion with clang and boost 1.73.0

```
/home/omv/freeorion/BUILD/freeorion-0.4.10/UI/FleetWnd.cpp:2416:68: error: use of undeclared identifier '_1'
        boost::bind(&FleetDetailPanel::ShipSelectionChanged, this, _1));
        |       |       |       |       |       |       |       |  ^
/home/omv/freeorion/BUILD/freeorion-0.4.10/UI/FleetWnd.cpp:2418:64: error: use of undeclared identifier '_1'
        boost::bind(&FleetDetailPanel::ShipRightClicked, this, _1, _2, _3));
        |       |       |       |       |       |       |      ^
/home/omv/freeorion/BUILD/freeorion-0.4.10/UI/FleetWnd.cpp:2418:68: error: use of undeclared identifier '_2'
        boost::bind(&FleetDetailPanel::ShipRightClicked, this, _1, _2, _3));
        |       |       |       |       |       |       |       |  ^
/home/omv/freeorion/BUILD/freeorion-0.4.10/UI/FleetWnd.cpp:2418:72: error: use of undeclared identifier '_3'
        boost::bind(&FleetDetailPanel::ShipRightClicked, this, _1, _2, _3));
        |       |       |       |       |       |       |       |      ^
/home/omv/freeorion/BUILD/freeorion-0.4.10/UI/FleetWnd.cpp:2420:69: error: use of undeclared identifier '_1'
        boost::bind(&FleetDetailPanel::UniverseObjectDeleted, this, _1));
        |       |       |       |       |       |       |       |   ^
/home/omv/freeorion/BUILD/freeorion-0.4.10/UI/FleetWnd.cpp:2819:63: error: use of undeclared identifier '_1'
        boost::bind(&FleetWnd::CreateNewFleetFromDrops, this, _1));
        |       |       |       |       |       |       |     ^
/home/omv/freeorion/BUILD/freeorion-0.4.10/UI/FleetWnd.cpp:2822:61: error: use of undeclared identifier '_1'
        boost::bind(&FleetWnd::UniverseObjectDeleted, this, _1));
```

--- a/UI/FleetWnd.cpp
+++ b/UI/FleetWnd.cpp
@@ -2412,12 +2412,14 @@ FleetDetailPanel::FleetDetailPanel(GG::X w, GG::Y h, int fleet_id, bool order_is
         m_ships_lb->AllowDropType(SHIP_DROP_TYPE_STRING);
     }
 
+    namespace ph = boost::placeholders;
+
     m_ships_lb->SelRowsChangedSignal.connect(
-        boost::bind(&FleetDetailPanel::ShipSelectionChanged, this, _1));
+        boost::bind(&FleetDetailPanel::ShipSelectionChanged, this, ph::_1));
     m_ships_lb->RightClickedRowSignal.connect(
-        boost::bind(&FleetDetailPanel::ShipRightClicked, this, _1, _2, _3));
+        boost::bind(&FleetDetailPanel::ShipRightClicked, this, ph::_1, ph::_2, ph::_3));
     GetUniverse().UniverseObjectDeleteSignal.connect(
-        boost::bind(&FleetDetailPanel::UniverseObjectDeleted, this, _1));
+        boost::bind(&FleetDetailPanel::UniverseObjectDeleted, this, ph::_1));
 }
 
 void FleetDetailPanel::CompleteConstruction() {
@@ -2811,6 +2813,8 @@ void FleetWnd::CompleteConstruction() {
 
     // determine fleets to show and populate list
     Refresh();
+
+    using boost::placeholders::_1;
 
     // create drop target
     m_new_fleet_drop_target = GG::Wnd::Create<FleetDataPanel>(GG::X1, ListRowHeight(), m_system_id, true);

--- a/UI/ObjectListWnd.cpp
+++ b/UI/ObjectListWnd.cpp
@@ -1902,6 +1902,8 @@ public:
         m_header_row = GG::Wnd::Create<ObjectHeaderRow>(GG::X1, ListRowHeight());
         SetColHeaders(m_header_row); // Gives ownership
 
+        using boost::placeholders::_1;
+
         m_header_row->ColumnsChangedSignal.connect(
             boost::bind(&ObjectListBox::Refresh, this));
         m_header_row->ColumnHeaderLeftClickSignal.connect(

